modules/macstadium_infrastructure/wjb.tf

```diff
@@ -44,5 +44,30 @@ resource "vsphere_virtual_machine" "wjb" {
   }
 }
 
+resource "null_resource" "worker" {
+  triggers {
+    ssh_key_file_signature = "${sha256(file(var.vm_ssh_key_path))}"
+  }
+
+  connection {
+    host = "${vsphere_virtual_machine.wjb.network_interface.0.ipv4_address}"
+    user = "${var.ssh_user}"
+    agent = true
+  }
+
+  provisioner "file" {
+    source      = "${var.vm_ssh_key_path}"
+    destination = "/tmp/travis-vm-ssh-key"
+  }
+
+  provisioner "remote-exec" {
+    inline = [
+      "sudo mv /tmp/travis-vm-ssh-key /etc/travis-vm-ssh-key",
+      "sudo chown travis-worker:travis-worker /etc/travis-vm-ssh-key",
```

Comment:
Shoot, this is going to break on new provisions since that user doesn't exist yet :thinking: